example-mojaloop-backend/Chart.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
       - dependency
       - backend
       - kafka
-    version: 26.8.5
+    version: 19.0.2
```

Comment:
Reverting to `19.0.2` as `26.8.5` has incompatibility issues that require investigating.